ghost/mailgun-client/lib/mailgun-client.js

```diff
@@ -172,12 +172,19 @@ module.exports = class MailgunClient {
         const providerId = event?.message?.headers['message-id'];
 
         return {
+            id: event.id,
             type: event.event,
             severity: event.severity,
             recipientEmail: event.recipient,
             emailId: event['user-variables'] && event['user-variables']['email-id'],
             providerId: providerId,
-            timestamp: new Date(event.timestamp * 1000)
+            timestamp: new Date(event.timestamp * 1000),
+
+            error: event['delivery-status'] && event['delivery-status'] ? {
```

Comment:
> event['delivery-status'] && event['delivery-status']

This seems like an accidental repeat, guess the second one wanted to test something like if `code` exists on `delivery-status` ?

Reply:
Oh good catch, that was certainly the intention